iac-extensions/terraform/src/test/resources/checks/ClearTextProtocolsCheck/test_ecs_task_definition.tf

```diff
@@ -3,13 +3,15 @@ resource "aws_ecs_task_definition" "ecs_task" {
   volume {
     # Noncompliant@+1 {{Omitting transit_encryption enables clear-text traffic. Make sure it is safe here.}}
     efs_volume_configuration {
+   #^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Minor: it doesn't look like this test case is in the scope of [SONARIAC-286](https://jira.sonarsource.com/browse/SONARIAC-286)